WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md

```diff
@@ -0,0 +1,401 @@
+
+The following options were added to Certutil:
+
+| Syntax | Description | Example |
+|--------|---------|---------|
+| CertUtil [Options] -syncWithWU DestinationDir| Sync with Windows Update.</br></br>- DestinationDir is the folder that receives the files by using the automatic update mechanism.</br>- The following files are downloaded by using the automatic update mechanism:</br></br> - The `authrootstl.cab` contains the CTLs of non-Microsoft root certificates.</br> - The `disallowedcertstl.cab` contains the CTLs of untrusted certificates.</br>- The `disallowedcert.sst` contains the serialized certificate store, including the untrusted certificates.</br>- The `thumbprint.crt` contains the non-Microsoft root certificates. | CertUtil -syncWithWU \\server1\PKI\CTLs |
+| CertUtil [Options] -generateSSTFromWU SSTFile | Generate SST by using the automatic update mechanism.</br></br>SSTFile: `.sst` file to be created. The generated .sst file contains the non_Microsoft root certificates that were downloaded by using the automatic update mechanism. | CertUtil –generateSSTFromWU TRoots.sst |
+
+> [!TIP]
+> `Certutil -SyncWithWU -f <folder>` updates existing files in the target folder.
+> `Certutil -syncWithWU -f -f <folder>` removes and replaces files in the target folder.
+
+## Potential errors with Certutil -SyncWithWU
+
+You may encounter the following errors and warnings when running the `Certutil -syncWithWU` command:
+
+- If you use a non-existent local path or folder as the destination folder, you'll see the error:
+  
+    The system can't find the file specified. 0x80070002 (WIN32: 2 ERROR\_FILE\_NOT\_FOUND)
+
+- If you use a non-existent or unavailable network location as the destination folder, you'll see the error:
+
+    The network name can't be found. 0x80070043 (WIN32: 67 ERROR\_BAD\_NET\_NAME)
+
+- If your server can't connect over TCP port 80 to Microsoft Automatic Update servers, you'll receive the following error:
+
+    A connection with the server couldn't be established 0x80072efd (INet: 12029 ERROR\_INTERNET\_CANNOT\_CONNECT)
+
+- If your server is unable to reach the Microsoft Automatic Update servers with the DNS name ctldl.windowsupdate.com, you'll receive the following error:
+
+    The server name or address couldn't be resolved 0x80072ee7 (INet: 12007 ERROR\_INTERNET\_NAME\_NOT\_RESOLVED).
+
+- If you don't use the **-f** switch, and any of the CTL files already exist in the directory, you'll receive a *file* exists error:
+
+    CertUtil: -syncWithWU command FAILED: 0x800700b7 (WIN32/HTTP: 183 ERROR\_ALREADY\_EXISTS) Certutil: Can't create a file when that file already exists.
+
+- If there's a change in the trusted root certificates, you'll see: "Warning\! Encountered the following no longer trusted roots: \<folder path\>\\\<thumbprint\>.crt. Use "-f -f" options to force the delete of the above ".crt" files. Was "authrootstl.cab" updated? If yes, consider deferring the delete until all clients have been updated."
```

Comment:
`Use "-f -f" options to force the delete of the above ".crt" files.` Can we replace or provide more locational information to the word 'above'. I'm not even sure the word above needs to be there?